validate/src/main/java/io/spine/validate/option/RangedConstraint.java

```diff
@@ -82,8 +82,8 @@ private static void verifyType(FieldDeclaration field, Range<ComparableNumber> r
     }
 
     private static void checkInteger(ComparableNumber number, FieldDeclaration field) {
-        checkState(number.isInteger(),
-                   "An integer bound expected for field `%s`, but got `%s`.", field, number);
+        Preconditions.checkState(number.isInteger(),
```

Comment:
Please see if we could keep this static import.

I think this might be IDEA "taking care" of us after the file movement. And it looks like it may not be just here :(